src/main/java/org/terasology/Lakes/LavaProvider.java

```diff
@@ -56,7 +56,7 @@ public void process(GeneratingRegion region) {
 
             for (Vector3i pos : processRegion) {
 
-                float noiseValue = noise.noise(pos.x(),pos.y(),pos.z());
+                float noiseValue = noise.noise(pos.x() * 0.1f,pos.y() * 0.1f,pos.z());
```

Comment:
What is the factor `0.1f` used for here? Is this scaling the noise to make the lakes a bit bigger?

Reply:
this is used to control the rarity of lakes. Lava Lakes were a bot too common